src/main/webapp/index.html

```diff
@@ -11,5 +11,7 @@
   <body>
     <div id="main">
```

Comment:
For the sake of a demo, I think this is good, but it would be nice to make these divs with a template as well and render on the dom root element.

Reply:
Good idea! I think we could work on that this week for the MVP.